### PR TITLE
Add eventId to build schema

### PR DESCRIPTION
**Pull request by tkyi**
Builds should know which event they are associated with; updating build schema to include an `eventId` when a GET request is made 

Related to https://github.com/screwdriver-cd/screwdriver/issues/221

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# CHANGELOG
+
+## 15.0.0
+
+Breaking changes:
+  * `eventId` should be returned when a GET request is made on a build.
```
Comment (minz1027) [inline CHANGELOG.md]: Nice!

File: models/build.js [hunk 2]
```diff
@@ -26,9 +26,14 @@ const MODEL = {
         .description('Identifier of this build')
         .example('4b8d9b530d2e5e297b4f470d5b0a6e1310d29c5e'),
 
+    eventId: Joi
```
Comment (minz1027) [inline models/build.js]: this should be added to the indexes, and add one more null to the rangeKeys

Comment (tkyi) [inline models/build.js]: added

**Comment by minz1027** [comment]
Tiff, can you squash the commits? The rest LGTM! 👍

**Pull request merged**